### PR TITLE
Documenting context property I missed in the initial update to Office…

### PR DESCRIPTION
**Pull request by eprofeta**
…365 mail adapter.

### CODE REVIEW

File: biztalk/core/office365-mail-adapter.md [hunk 1]
```diff
@@ -63,18 +63,22 @@ The Office 365 Outlook Email Adapter allows you to send and receive mails from y
...
 
-|Property|Promoted property|
-|---|---|
-| **To** | OfficeMail.To |
-| **CC** | OfficeMail.CC |
-| **Subject** | OfficeMail.Subject |
-| **Importance** | OfficeMail.Importance |
+    Property|Promoted property|
+    |---|---|
+    | **To** | OfficeMail&#46;To |
```
Comment (msnehagup) [inline biztalk/core/office365-mail-adapter.md]: can you please revert the ascii changes: `.` -> `&#46;`,  ` ` -> `&nbsp;`

File: biztalk/core/office365-mail-adapter.md [hunk 2]
```diff
@@ -63,18 +63,22 @@ The Office 365 Outlook Email Adapter allows you to send and receive mails from y
...
+    | **CC** | OfficeMail&#46;CC |
+    | **Subject** | OfficeMail.Subject |
+    | **Importance** | OfficeMail.Importance |
+    | **Attached&nbsp;Files** | OfficeMail.AttachedFiles |
+
+
+2. The *OfficeMail.AttachedFiles* property value is list of files with "|" as delimiter in between different files. For instance: "C:\\Attachments\File1.pdf|C:\\Attachments\File2.xml".
```
Comment (msnehagup) [inline biztalk/core/office365-mail-adapter.md]: when you're quoting property names, either use bold (**OfficeMail.AttachedFiles**) or back ticks (`OfficeMail.AttachedFiles`)

Comment (eprofeta) [reply biztalk/core/office365-mail-adapter.md]: done

File: biztalk/core/office365-mail-adapter.md (hunk 1 above)
Comment (eprofeta) [reply biztalk/core/office365-mail-adapter.md]: if I leave the ., it shows hyperlinked in the UI. I don't want a hyperlink.

Comment (msnehagup) [reply biztalk/core/office365-mail-adapter.md]: worked fine for me. Subject and Importance have . , how is it working there?

Comment (eprofeta) [reply biztalk/core/office365-mail-adapter.md]: Fixed.

**Review by msnehagup — APPROVED** [review]

**Pull request merged**